src/Services/organizationService.js

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+
+const getOrganizacion = async () => {
```

Comment:
Please write everything using only English. In this case it should be getOrganization.
Same applies to getOrganizacionById